### PR TITLE
pip-tools

### PR DESCRIPTION
**Pull request by Br4nd0R**
# Changes
* Requirements now reside in two files: `requirements/dev.in` and `requirements/prod.in`
* Two new make targets for updating and installing dependencies: `update-deps` & `dev-install`
* I've pinned the following libraries:
    * `django==3.1.8`
    * `djangorestframework-filter==1.0.0.dev2`

### CODE REVIEW

File: requirements/prod.in [hunk 1]
```diff
@@ -0,0 +1,35 @@
+ansi2html
+boto3
+coreapi
+cumulusci
+django==3.1.8
```
Comment (davisagli) [inline requirements/prod.in:5]:
```suggestion
django<3.2
```

File: requirements/prod.in [hunk 2]
```diff
@@ -0,0 +1,35 @@
+ansi2html
+boto3
+coreapi
```
Comment (davisagli) [inline requirements/prod.in:3]: We don't appear to be using coreapi

File: requirements/prod.in [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+django-redis
+django-rq
+django-slds-crispyforms
+django-storages
+django-watson
+djangorestframework
+djangorestframework-filters==1.0.0.dev2
+django-filter
+gunicorn
+honcho>=0.7.1
```
Comment (davisagli) [inline requirements/prod.in:24]:
```suggestion
honcho
```

File: requirements/prod.in [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+junitparser>=0.8
```
Comment (davisagli) [inline requirements/prod.in:25]:
```suggestion
```

We don't appear to be using this.

**Review by davisagli — CHANGES_REQUESTED** [review]

File: requirements/prod.in [hunk 5]
```diff
@@ -0,0 +1,34 @@
+ansi2html
+boto3
+coreapi
```
Comment (Br4nd0R) [inline requirements/prod.in:3]:
```suggestion
```

**Review by davisagli — APPROVED** [review]

**Pull request merged**